protocols/rhinofi.json

```diff
@@ -0,0 +1,54 @@
+{
+	"name": "rhino.fi",
+	"socials": [
+		{
+			"label": "Website",
+			"link": "https://rhino.fi/"
+		},
+		{
+			"label": "Discord",
+			"link": "https://discord.gg/yafEx8W"
+		},
+		{
+			"label": "Blog",
+			"link": "https://rhino.fi/blog/"
+		},
+		{
+			"label": "Reddit",
+			"link": "https://www.reddit.com/r/rhino.fi/"
+		},
+		{
+			"label": "Twitter",
+			"link": "https://x.com/rhinofi"
+		},
+		{
+			"label": "Github",
+			"link": "https://github.com/rhinofi"
+		},
+		{
+			"label": "Docs",
+			"link": "https://tech.rhino.fi/rhino.fi"
+		}
+	],
+	"logo_dark": "/protocols/rhinofi-logo-dark.svg",
+	"logo_light": "/protocols/rhinofi-logo-light.svg",
+	"icon_url": "/protocols/rhinofi-icon-dark.png",
+	"icon_dark": "/protocols/rhinofi-icon-dark.png",
+	"icon_light": "/protocols/rhinofi-icon-light.png",
+	"description": "rhino.fi is a Layer 2 bridge for effortless multi-chain token movement",
+	"mainnet": "yes",
+	"type": "bridge",
+	"chains_supported": "Ethereum, Arbitrum, BSC, Polygon, zkSync, zkEVM, Optimism, Linea, Base, Manta, Starknet, Scroll",
```

Comment:
I think a few are missing here?

Reply:
Ethereum is a good point because the way deposits work from there is quite different as it links in to the StarkEx contract